modules/pyqt5.yaml

```diff
@@ -0,0 +1,51 @@
+name: pyqt5
```

Comment:
we have a base app for pyqt5 that allow not having to build it.

Reply:
Hmm, that would lock me into whatever runtime the pyqt5 app uses, right? That would be a bit annoying as it would hinder the Qt6 porting effort (the Python stuff is separate from the C++ stuff, so the latter can be ported to Qt6 first, while people can still make use of Qt5 in their custom Python modules).
Not having to build PyQt would be very nice though, especially since it currently is only used by one of the modules that we ship with by default.

Reply:
not it wouldn't. there is a version for Qt6 as well.

Reply:
That's really nice, thanks for the hint!